app/controllers/users_controller.rb

```diff
@@ -113,6 +114,30 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def change_password
+    if request.get?
+      puts "XXX we've got a GET request"
+    end
+    puts "XXX change password called"
+  end
+
+  def update_password
```

Comment:
How so? Sorry this was my first ruby change ever :) so pardon my ignorance but I don't understand what the code would look like?